katello/hooks/boot/01-helpers.rb

```diff
@@ -16,6 +16,11 @@ def log_and_say(level, message, do_say = true, do_log = true)
       Kafo::KafoConfigure.logger.send(level, message) if do_log
     end
 
+    def fail_and_exit(message)
```

Comment:
If you move this to `HookContextExtension` you don't need to prefix it in the actual hooks.

Reply:
The function it calls, log_and_say is already in the helper so I wanted to co-locate them for the short term. Based on https://github.com/theforeman/foreman-installer/pull/407#discussion_r349763345 and my own observations, dropping the Helper for functions that are added via an extension as a separate re-factor would be a good clean up. And to provide some parity with Foreman hooks.

I would love to tackle that after this. Trying not to overload this particular change.

Reply:
Fair enough. You can consider adding them in both places but this is already getting large.